### ISSUE TITLE
Indicate file loading

### ISSUE THREAD

**Issue by martin-desktops** [knuxify/eartag#36]
With bigger music folders, it might take a while to load all the library. During this process, the app doesn't behave like something is in progress. To avoid the feeling that something is not right, I think we should give some feedback to the user. Here's how it was solved in the Amberol app.

https://user-images.githubusercontent.com/64419373/190911286-6b07c8af-00f7-4825-b7ad-6dc235c0a596.mp4

I also want to thanks to all contributors for such an elegant and functional app. 💛

**Comment by knuxify** [comment]
Technically you're not supposed to drag your entire library into Ear Tag, the multiple file view is mostly there so that you can edit multiple files at once...

Still, this is pretty useful for e.g. larger music files, or files on network shares, where the loading process might take a bit longer, and certainly something to work on. Thanks for the suggestion!

**Comment by martin-desktops** [comment]
Yeah, who would think that importing 10GB of music will take a while... More practical example on my hardware - for one folder, 200MB in total it takes for Ear Tag 8 seconds to load the whole folder. The indicator would be definitely helpful!
